app/config/AppConfig.scala

```diff
@@ -46,6 +49,24 @@ class AppConfig @Inject() (val config: Configuration, servicesConfig: ServicesCo
   val upscanMaximumFileSize: Long =
     config.get[ConfigMemorySize]("microservice.services.upscan-initiate.maximum-file-size").toBytes
 
-  val sdesUrl: String          = servicesConfig.baseUrl("sdes")
-  val sdesFilereadyUri: String = config.get[String]("microservice.services.sdes.uri")
+  // SDES
+  lazy val useProxy: Boolean = config.get[Boolean]("sdes.use-proxy")
+  lazy val service: String   = if (useProxy) "secure-data-exchange-proxy" else "sdes-stub"
+
+  lazy val sdesInformationType: String = config.get[String]("sdes.information-type")
+  lazy val sdesSrn: String             = config.get[String]("sdes.srn")
+  lazy val sdesClientId: String        = config.get[String]("sdes.client-id")
+
+  // SDES Proxy
+
+  lazy val sdesUrl: AbsoluteUrl =
+    AbsoluteUrl.parse(servicesConfig.baseUrl(service))
+  lazy val sdesFileReadyUri: UrlPath =
+    UrlPath(
+      config
+        .get[String](s"microservice.services.$service.file-ready-uri")
```

Comment:
file-ready-uri only exists for the proxy, might be worth updating it to uri so it matches the app config PR

Reply:
Ah since the config has been updated, will need to update the stub to file-ready-uri instead! :)